frontend/packages/gitops-plugin/src/components/list/GitOpsTableRow.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import i18n from '@console/internal/i18n';
+import * as classNames from 'classnames';
+import { GitOpsAppGroupData } from '../utils/gitops-types';
+import { RowFunction, TableData, TableRow } from '@console/internal/components/factory';
+import { Button } from '@patternfly/react-core';
+import { history, ExternalLink } from '@console/internal/components/utils';
+import { routeDecoratorIcon } from '@console/dev-console/src/components/import/render-utils';
+
+const tableColumnClasses = [
+  classNames('pf-m-width-20'), // Application name
+  classNames('pf-m-width-40'), // Git repository
+  classNames('pf-m-hidden', 'pf-m-visible-on-md', 'pf-m-width-20'), // Environments
+  classNames('pf-m-hidden', 'pf-m-visible-on-lg', 'pf-m-width-20'), // Last deployment
+];
+
+const handleClick = (appGroup) => {
```

Comment:
Type standalone functions the best you can...

```suggestion
const handleClick = (appGroup: GitOpsAppGroupData) => {
```